website/components/features-list/index.jsx

```diff
@@ -0,0 +1,22 @@
+import Feature from './feature'
+import s from './style.module.css'
+
+export default function FeaturesList({ title, features }) {
+  return (
+    <div
+      className={s.featureListContainer}
+      style={{
+        backgroundImage: `url(${require('./images/top-right-design.svg')}), url(${require('./images/bottom-left-design.svg')})`,
```

Comment:
Nice love how you're passing this here, and including the images in this component.  Nice encapsulation 👌